### PR TITLE
Small fixes to cb_info table

### PR DESCRIPTION
**Pull request by mtmcgrew**
* dynamically pull the correct windows directory if it's not c:\windows
* fix a compile warning 
* replace a %20 with a space in sensor group names
* documentation fix

### CODE REVIEW

File: osquery/tables/applications/posix/carbon_black.cpp [hunk 1]
```diff
@@ -88,8 +90,8 @@ void getQueue(Row& r) {
   if (!listFilesInDirectory(kCbDir, files_list, true)) {
     return;
   }
-  unsigned int binary_queue_size = 0;
-  unsigned int event_queue_size = 0;
+  uintmax_t binary_queue_size = 0;
```
Comment (theopolis) [inline osquery/tables/applications/posix/carbon_black.cpp]: I think a `std::size_t` works too, but best to go with what the API returns. Nice.

File: osquery/tables/applications/windows/carbon_black.cpp [hunk 2]
```diff
@@ -17,7 +17,9 @@
 #include <osquery/filesystem.h>
 #include <osquery/tables.h>
 
+#include "Shlwapi.h"
```
Comment (theopolis) [inline osquery/tables/applications/windows/carbon_black.cpp]: Place these OS includes above boost's in their own category.

File: osquery/tables/applications/windows/carbon_black.cpp [hunk 3]
```diff
@@ -26,17 +28,18 @@ namespace tables {
 
 // Carbon Black registry path
 #define kCbRegLoc "SOFTWARE\\CarbonBlack\\config"
-// Path to Carbon Black direcotry
-#define kCbDir "C:\\Windows\\CarbonBlack\\"
 
 void getQueue(Row& r) {
+  char windowsPath[MAX_PATH];
+  GetWindowsDirectory(windowsPath, MAX_PATH);
```
Comment (theopolis) [inline osquery/tables/applications/windows/carbon_black.cpp]: I think this is OK. Maybe we need a better abstraction for this, returned as a `fs::path` so you can then:
```cpp
fs::path root = getWindowsRoot();
root /= "CarbonBlack"
```

/cc @yying @PoppySeedPlehzr

**Comment by osqueryer** [comment]
:-1: The commit 290adfecd84293bd8b2f0dfa860bb78019216feb (Job results: [3089](https://jenkins.osquery.io/job/osqueryPullRequestBuild/3089/)) failed one or more tests (**OS X/Linux**).

File: osquery/tables/applications/windows/carbon_black.cpp (hunk 3 above)
Comment (PoppySeedPlehzr) [inline osquery/tables/applications/windows/carbon_black.cpp]: I have an PR inbound, that contains a new fn in filesystem, `getSystemRoot()` that will return an fs::path.

**Comment by theopolis** [comment]
Ok if you `git pull --rebase origin master` this should pass.

**Comment by theopolis** [comment]
But you're still waiting on @PoppySeedPlehzr's commit, got it.

**Comment by PoppySeedPlehzr** [comment]
The commit adding the GetSystemRoot function has been landed, and the DWORD bugs fixed. If you rebase you should be gtg!

**Comment by osqueryer** [comment]
:no_good: The commit 32fbca7cca3c2b13e2424d7bd7556899e76c0369 (Job results: [91](https://jenkins.osquery.io/job/osqueryPullRequestAudit/91/)) failed the code audit.

**Comment by osqueryer** [comment]
:no_good: The commit aee8cb311a599cbaafb135e088716683f3011646 (Job results: [92](https://jenkins.osquery.io/job/osqueryPullRequestAudit/92/)) failed the code audit.

**Comment by osqueryer** [comment]
:-1: The commit aee8cb311a599cbaafb135e088716683f3011646 (Job results: [3109](https://jenkins.osquery.io/job/osqueryPullRequestBuild/3109/)) failed one or more tests (**OS X/Linux**).

**Pull request closed without merging**